smoketest.bash

```diff
@@ -205,6 +208,18 @@ if [ "${USE_PROXY}" = "true" ]; then
     createProxyCfgVolume
 fi
 
+createProxyCertsVolume() {
+    "${container_engine}" volume create auth_proxy_certs
+    "${container_engine}" container create --name proxy_certs_helper -v auth_proxy_certs:/certs busybox
+    chmod 444 "${DIR}/compose/auth_certs/private.key"
+    chmod 444 "${DIR}/compose/auth_certs/certificate.pem"
```

Comment:
Maybe the `chmod` can be done after the `cp` so that the permissions of the copy within the container are what changes, rather than the permissions of the file on the host machine?

Also, if these files don't exist yet this will fail, right? It might be nice to put a check for the file existence before the `chmod` or `podman cp` and exit with a nicer error message if they don't exist yet. A note in the README section about smoketesting might also be helpful if this is a new prerequisite.

Reply:
> Maybe the chmod can be done after the cp

And by this I mean doing something like `podman exec proxy_certs_helper chmod 444 /certs/certificate.pem`, not just moving the line below the `podman cp`

Reply:
I don't think I can move the `chmod` after the `cp` because the container isn't actually up and running?

Reply:
True. Instead of `podman exec`, would `podman run --entrypoint=/bin/sh` work?

Reply:
hmm I can't run a container that has already been created, `podman start` doesn't have `--entrypoint` or `-c` flags, and I can't run `podman run` instead of `podman create` because I need the files to be copied in before running the `chmod` command.